thefuck/rules/go_unknown_command.py

```diff
@@ -0,0 +1,18 @@
+from thefuck.utils import get_closest, replace_argument
+
+_GOLANG_COMMANDS = (
+    "bug", "build", "clean", "doc", "env", "fix", "fmt", "generate", "get",
+    "install", "list", "mod", "run", "test", "tool", "version", "vet")
+
+
+def match(command):
+    return (command.script_parts
+            and command.script_parts[0] == "go"
```

Comment:
It will be better to replace that check with `for_app` decorator, it helps with things like `sudo`:
```python
from thefuck.utils import for_app

@for_app('go')
def match(command):
    ...
```